jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostClassifier.scala

```diff
@@ -292,7 +292,9 @@ class XGBoostClassificationModel private[ml](
 
         private val batchIterImpl = rowIterator.grouped($(inferBatchSize)).flatMap { batchRow =>
           if (batchCnt == 0) {
-            val rabitEnv = Array("DMLC_TASK_ID" -> TaskContext.getPartitionId().toString).toMap
+            val rabitEnv = Array(
+              "DMLC_TASK_ID" -> TaskContext.getPartitionId().toString,
+              "DMLC_WORKER_STOP_PROCESS_ON_ERROR" -> "false").toMap
```

Comment:
`DMLC_WORKER_STOP_PROCESS_ON_ERROR`,  what's the meaning of this tag?

Reply:
it's newly added in https://github.com/dmlc/rabit/blob/65b718a5e786bd7d0a850f3fa1df0dbdab023eb1/src/allreduce_base.cc